app/src/main/java/com/randomappsinc/studentpicker/extentions/ViewExt.kt

```diff
@@ -0,0 +1,7 @@
+package com.randomappsinc.studentpicker.extentions
+
+import android.view.View
+
+fun View.maybeVisible(show: Boolean) {
```

Comment:
Ah, that's how maybeVisible works. I don't think this extension is worth it, and extensions on core classes can make them less readable over time. It's also not clear from the API name what it does - What if we want the not visible state to be View.INVISIBLE instead?